nodeServer/ts/ZssServerConnection.ts

```diff
@@ -11,6 +11,7 @@
 import { FTAConnectionTarget, FTAFileInfo, FTAFileAttributes, LinuxStatsMask, FTASide } from "../../common/FTATypes";
 import { FTAConnection, FTAErrorHandler, FTAConnectionEvents } from "./FTAConnection";
 
+let httpEndpoint = '/unixfile/contents';
```

Comment:
You should be able to get to this more reliably by using "callRootService", https://github.com/zowe/zss-auth/blob/master/lib/zssAuth.js#L38 but let me know if there's a reason why that isnt a better choice.

Reply:
Nevermind this - the structure of this whole service is very unlike other services we have so it doesn't really apply at this time. What the file transfer app services are doing is establishing a websocket to minimize the amount of requests that the client sends, but this has the consequence of needing the server to have some custom proxying logic which is not at all elegant when the server was already built to proxy under the hood if you were to ask for certain APIs (which are easy to get to via uriBroker in the web). So, I'd say at some point we need to evaluate if this code needs to be rewritten, or if we need to provide conveniences for websockets and promote this more, but we can skip this for now.